onset_maproom/config-sample.yaml

```diff
@@ -4,7 +4,15 @@ core_path: /onset-maproom
 admin_path: /onset-maproom-admin
 tile_path: /onset-maproom-tiles
 daily_rainfall_path: /data/aaron/ethiopia-rain-rechunked
+daily_tmin_path: /data/remic/mydatafiles/ANACIM/anacim_v3/ALL_block2/tn_mrg_daily
+daily_tmax_path: /data/remic/mydatafiles/ANACIM/anacim_v3/ALL_block2/tx_mrg_daily
```

Comment:
They are used by enactstmaxtozarr.py and enactstmintozarr.py
It's similar problem discussed in other PR that we have data curation, calculus and marpooms all in same place.

Reply:
`daily_tmax_path` does not occur in `enactstmaxtozarr.py`. It uses `tmax_mrg_nc_path` and `tmax_mrg_zarr_path`. Likewise for tmin.

Reply:
I suggest you remove them and add back when they're needed, but I won't insist on it. Resolve this conversation whenever you're ready to.

Reply:
Resolved this with a commit removing those lines. This is all going to have to be revised when rebasing on master anyway since we now have 2 config files